### PR TITLE
[docs] [integrations] update integrations page on kairosdb  

### LINKED ISSUE #11172: [Docs] The KairosDB integrations page needs detailed instructions
### Description

The current page needs to be more explicit. Adding some suggestions by @ddhodge as well to the list:
- Include a prerequisites section to mention which Java version and so on. 
- The Kairos version install instructions can be replaced by a [link](https://kairosdb.github.io/docs/GettingStarted.html) to the kairos page instructions rather than having to use and update the wget command.
- The  push and query instructions should have a step by step way with results/output.
- Steps to connect to a non-local cluster ( Mentioned by @Anagha-Damodaran )

### PR DESCRIPTION
**Pull request by aishwarya24**
resolves #11172 

@netlify /latest/integrations/kairosdb/

### CODE REVIEW

File: docs/content/latest/integrations/kairosdb.md [hunk 1]
```diff
@@ -73,32 +70,139 @@ Use HELP for help.
...
 
-## 5. Test KairosDB
+## Test KairosDB
+
+- Start Postman and create a new workspace.
 
 ### Push data
 
-Push metric data into KairosDB as per the instructions [here](https://kairosdb.github.io/docs/PushingData.html).
+- Select the POST API request and add the following URL in the `Enter request URL` box:
```
Comment (ddhodge) [inline docs/content/latest/integrations/kairosdb.md:102]: "Select the POST API request" - can you be more specific? Do you click some button, or other UI object?

File: docs/content/latest/integrations/kairosdb.md [hunk 2]
```diff
@@ -73,32 +70,139 @@ Use HELP for help.
...
+
+```text
+http://localhost:8080/api/v1/datapoints
+```
+
+- In the body of the request, add the following JSON, and send it.
```
Comment (ddhodge) [inline docs/content/latest/integrations/kairosdb.md:108]:
```suggestion
- In the body of the request, add the following JSON, then click **Send**.
```

File: docs/content/latest/integrations/kairosdb.md [hunk 3]
```diff
@@ -73,32 +70,139 @@ Use HELP for help.
...
+      "tags": {
+          "host": "server2"
+      }
+  }
+]
+```
+
+Your request should look like:
+
+![kairosdb POST request](/images/develop/ecosystem-integrations/kairosdb/kairosdb1.png)
```
Comment (ddhodge) [inline docs/content/latest/integrations/kairosdb.md:134]: Possible to resize the windows of these screenshots so they doesn't show all that whitespace? They would then be easier to read, being less wide.

File: docs/content/latest/integrations/kairosdb.md [hunk 4]
```diff
@@ -14,40 +14,37 @@ showAsideToc: true
...
+## Prerequisites
 
-Follow [Quick start](../../quick-start/) to run a local YugabyteDB cluster and test YugabyteDB's YCQL API to confirm that you have a YCQL service running on `localhost:9042`.
+Before you start using the KairosDB plugin, ensure that you have:
 
-## 2. Download KairosDB
+- Java version 1.8 or later.
+- The latest version of [KairosDB](https://kairosdb.github.io/docs/GettingStarted.html).
+- The latest version of [YugabyteDB plugin for KairosDB](https://github.com/yugabyte/kairosdb-yb-plugin/releases/) and explore the [README] for details about the plugin.
+- A YugabyteDB cluster. Refer [YugabyteDB Quick start guide](/latest/quick-start/) to install and start a local cluster.
```
Comment (ddhodge) [inline docs/content/latest/integrations/kairosdb.md:24]:
```suggestion
- A YugabyteDB cluster. Refer to [YugabyteDB Quick start guide](/latest/quick-start/) to install and start a local cluster.
```

**Review by ddhodge — APPROVED** [review]
Misc comments

File: docs/content/latest/integrations/kairosdb.md (hunk 1 above)
Comment (aishwarya24) [reply docs/content/latest/integrations/kairosdb.md:102]: Added more steps and screenshots to be more clear.

File: docs/content/latest/integrations/kairosdb.md (hunk 3 above)
Comment (aishwarya24) [reply docs/content/latest/integrations/kairosdb.md:134]: Replaced it with the ones that showed on my Postman UI.

File: docs/content/latest/integrations/kairosdb.md [hunk 5]
```diff
@@ -73,32 +70,139 @@ Use HELP for help.
 ycqlsh>
 ```
 
-- Run the following YCQL commands to verify it is working:
+- Run the following YCQL commands, and connect to the `kairosdb` keyspace to verify it is working:
```
[the remark this answers is not in the document]
Comment (aishwarya24) [reply docs/content/latest/integrations/kairosdb.md:73]: I believe when we start kairosdb with the run command , it creates the database based on the configuration parameters we pass to the conf file. @Sfurti-yb should be able to confirm this.

[the remark this answers is not in the document]
Comment (Sfurti-yb) [reply docs/content/latest/integrations/kairosdb.md:73]: Yes the kairosdb keyspace is created when we use the run command

**Review by Sfurti-yb — APPROVED** [review]
LGTM

**Pull request merged**